modules/virt-about-deploying-templates-custom-namespace.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-about-deploying-templates-custom-namespace.adoc
+
+[id="virt-about-deploying-templates-custom-namespace_{context}"]
+= About deploying templates to a custom namespace
+
+To add templates to a custom namespace for private use, edit the `hco` custom resource (CR) and specify a custom namespace. Then, add the Red Hat templates to that custom namespace.
+
+[NOTE]
+====
+Only you can view and use the templates deployed to a custom namespace. Other users cannot view or use them.
```

Comment:
This needs to be clarified slightly. The admin must create the new namespace to be used, and would be responsible for any roles allowed in that namespace. This includes providing the ability for the SSP operator to deploy there.

Reply:
Thanks @sarahbx. I've updated this note to say:
When you create a private namespace as a cluster administrator, you can assign roles with permissions to enable other users to access that namespace and deploy templates to that namespace.